Docs/Book/Install.md

```diff
@@ -33,7 +33,40 @@ Windows users will use a slightly different command:
 
 ### How to build from source with Conda
 
-For more details on building from source with Conda, see the [conda-rdkit repository](https://github.com/rdkit/conda-rdkit)
+For more details on building from source with Conda, see the [conda-rdkit repository](https://github.com/rdkit/conda-rdkit).
+
+#### macOS 10.12 (Sierra): Python 3 environment
+
+The following commands will create a development environment for macOS Sierra and Python 3. Download
+Miniconda3-latest-MacOSX-x86_64.sh from [Conda](http://conda.pydata.org/miniconda.html) and run these
+following commands:
+   
+	bash Miniconda3-latest-MacOSX-x86_64.sh
+	conda install numpy scipy matplotlib
+	conda install cmake
+	conda install -c  https://conda.anaconda.org/rdkit  boost
+	conda install -c  https://conda.anaconda.org/rdkit  cairocffi
+	conda install pillow
+	conda install anaconda
+	conda install --channel https://conda.anaconda.org/rwest eigen3
+
+Optionally, add the following packages to your environment. 
+
+	pip install yapf==0.11.1
+	pip install coverage==3.7.1
+
+Then follow the usual build instructions. The PYTHON\_INCLUDE\_DIR must be set in the
+cmake command.
+
+	PYROOT=<path to miniconda3>
+	cmake -DPYTHON_INCLUDE_DIR=$PYROOT/include/python3.5m ..
```

Comment:
This is yours:
```
cmake -DPYTHON_EXECUTABLE=$PYROOT/bin/python  \
          -DRDK_BUILD_AVALON_SUPPORT=ON \
          -DRDK_BUILD_INCHI_SUPPORT=ON \
          -DRDK_BUILD_CAIRO_SUPPORT=ON -DRDK_BUILD_THREADSAFE_SSS=ON -DRDK_TEST_MULTITHREADED=ON \
	  -DPYTHON_LIBRARY=$PYROOT/lib/libpython3.5m.dylib \
	  -DPYTHON_INCLUDE_DIR=$PYROOT/include/python3.5m \
	  -DPYTHON_NUMPY_INCLUDE_PATH=$PYROOT/lib/python3.5/site-packages/numpy/core/include/ \
	  -DCMAKE_BUILD_TYPE=Release \
	  -DRDK_INSTALL_INTREE=OFF \
	  -DCMAKE_INSTALL_PREFIX=$INSTALL \
          -DBOOST_ROOT=$PYROOT \
          -DBoost_NO_BOOST_CMAKE=FALSE \
          -DBoost_NO_SYSTEM_PATHS=on \
          -DBoost_USE_STATIC_LIBS=off \
	  $SRCDIR 
```
I just checked what the settings are in the CMakeCache.txt file after running my command:

- `PYTHON_EXECUTABLE`: correctly found
- `PYTHON_LIBRARY`: correctly found
- `PYTHON_INCLUDE_DIR`: that is not correctly found and needs to be specified
- `PYTHON_NUMPY_INCLUDE_PATH`: correctly found
- `RDK_BUILD_AVALON_SUPPORT`: now added
- `RDK_BUILD_INCHI_SUPPORT`: now added
- `RDK_BUILD_CAIRO_SUPPORT`: now added 
- `RDK_BUILD_THREADSAFE_SSS`: on by default
- `RDK_TEST_MULTITHREADED`: on by default
- `CMAKE_BUILD_TYPE`: default
- `RDK_INSTALL_INTREE`: ON by default and that works for me
- `CMAKE_INSTALL_PREFIX`: not required for RDK_INSTALL_INTREE

Then there are the Boost settings. I cannot find these in the CMakeCache.txt file.

- `BOOST_ROOT`
- `Boost_NO_BOOST_CMAKE`
- `Boost_NO_SYSTEM_PATHS`
- `Boost_USE_STATIC_LIBS